src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts.mdx

```diff
@@ -64,11 +64,14 @@ These are the available notification channel types.
     id="user"
     title="User"
   >
-    For your convenience, we automatically load all users and their email addresses for the selected account. If your account has one or more [sub-accounts](/docs/accounts/install-new-relic/account-setup/manage-apps-or-users-sub-accounts), the notification channel includes only users for the [currently selected master or sub-account](/docs/accounts/install-new-relic/account-setup/use-multiple-accounts).
+    Requirements: 
+      * This feature is not available for users on the [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models), only to users on the original user model. As a workaround, you can use the [email notification channel](#email). 
 
     Use the **User** notification channel to select existing account team members and admins. To view the **Users** list or to [add users to alert policies](/docs/alerts/new-relic-alerts-beta/managing-notification-channels/view-or-update-user-email-channels): Go to **[one.newrelic.com](https://one.newrelic.com/ "Link opens in a new window.")**, in the top nav click **Alerts & AI**, then click **Notification channels**.
 
-    A user channel also sends [push notifications](#mobile-push) to any of the user's registered mobile devices. A device is registered if the user has logged into New Relic using the mobile app on the device.
+    If the user has [configured the New Relic mobile app](/docs/mobile-apps/new-relic-mobile-apps/authentication-alerts/alerting-new-relic-mobile-apps/), the user channel also sends [push notifications](#mobile-push) to any of the user's registered mobile devices. 
```

Comment:
I feel like you could be a little more consistent about using you/your or the user. I'm fine with either, but I'd love for it to be consistent. I feel a little thrown by the jump from "the user" to "your account" below. Or perhaps that I can't detect the internal reasons for using one or the other here?

Reply:
yeah it's a bit awkward because there's good chance they're adding notification channels for other people.